cmake/FindGoogleHash.cmake

```diff
@@ -18,6 +18,6 @@ if(GOOGLEHASH_INCLUDES)
 endif(GOOGLEHASH_INCLUDES)
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(GOOGLEHASH DEFAULT_MSG GOOGLEHASH_INCLUDES GOOGLEHASH_COMPILE)
+find_package_handle_standard_args(Googlehash DEFAULT_MSG GOOGLEHASH_INCLUDES GOOGLEHASH_COMPILE)
```

Comment:
this should have been GoogleHash -- it needs to match the case of the filename FindGoogleHash.cmake to fix the cmake warning